dev/src/pool.ts

```diff
@@ -202,22 +207,23 @@ export class ClientPool<T extends object> {
     }
 
     logger(
-      'ClientPool.release',
+      `ClientPool[${this.instanceId}].release`,
       requestTag,
       'Garbage collecting client [%s] (%s)',
       clientId,
       this.lazyLogStringForAllClientIds
     );
 
-    this.activeClients.delete(client);
+    const activeClientDeleted = this.activeClients.delete(client);
     this.failedClients.delete(client);
     await this.clientDestructor(client);
 
     logger(
-      'ClientPool.release',
+      `ClientPool[${this.instanceId}].release`,
       requestTag,
-      'Garbage collected client [%s] (%s)',
+      'Garbage collected client [%s] activeClientDeleted=%s (%s)',
```

Comment:
`activeClientDeleted=%s`

Double checking to ensure that %s will accept the boolean value and format it correctly?

Reply:
Yep, it does. Here is an example log message:

```
W9mwH [ClientPool[cplcNNlZ].release]: Garbage collected client [cliGq5Tk] activeClientDeleted=true (0 active clients: {}, 0 failed clients: {})
```

The message contains `activeClientDeleted=true` as expected.